projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/radio-button-docs/radio-button-docs.component.html

```diff
@@ -0,0 +1,193 @@
+      <p class="go-body-copy">
+        On the <code class="code-block--inline">&lt;go-radio-group&gt;</code>, the 
+        <code class="code-block--inline">legend</code> is the name of the set of radio options and
+        the <code class="code-block--inline">[control]</code> is the corresponding <code class="code-block--inline">FormControl</code>.
+      </p>
+      <p class="go-body-copy">
+        On the <code class="code-block--inline">&lt;go-radio-button&gt;</code>, the 
+        <code class="code-block--inline">label</code> is the name of that radio option and
+        the <code class="code-block--inline">formValue</code> is the value of that option.
+      </p>
+      <div class="go-container">
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">View</h2>
+          <go-radio-group legend="Options"
+                          [control]="radio1">
+            <div>
+              <go-radio-button label="Option 1"
+                               formValue="option1">
+              </go-radio-button>
+            </div>
+            <div>
+              <go-radio-button label="Option 2"
+                               formValue="option2">
+              </go-radio-button>
+            </div>
+          </go-radio-group>
+        </div>
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">Code</h2>
+          <code [highlight]="radio1Ex"
+                class="code-block--no-bottom-margin">
+          </code>
+        </div>
+      </div>
+    </ng-container>
+  </go-card>
+
+  <go-card id="form-radio-key" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h1 class="go-heading-5">Radio Keys</h1>
+    </ng-container>
+    <ng-container go-card-content>
+      <p class="go-body-copy">
+        The label attribute is used to generate a unique ID to associate the label with the radio button.
+        As this might not be desired, the <code class="code-block--inline">@Input() key: string;</code>
+        can be used to configure the ID attribute of the radio directly.
+        Anything passed into the component via the <code class="code-block--inline">[key]</code>
+        attribute will be used to both assign the ID to the radio and
+        associate the label with the radio button.
+      </p>
+      <div class="go-container">
+        <div class="go-column go-column--50">
+          <h2 class="go-heading-6 go-heading--underlined">View</h2>
+          <go-radio-group legend="Options"
+                          [control]="radio2">
+            <div>
+              <go-radio-button label="Option 1"
+                               formValue="option1"
+                               key="option_1">
+              </go-radio-button>
+            </div>
+            <div>
+              <go-radio-button label="Option 2"
+                               formValue="option2"
+                               key="option_1">
```

Comment:
Should this be `key="option_2"`?

Reply:
Yep, good call. I fixed this and the ts file that generated the code sample because it was wrong too.